sql/tasks.py

```diff
@@ -0,0 +1,122 @@
+from sqlalchemy import create_engine, extract
+from sqlalchemy.orm import sessionmaker
+from sql.entities import Task
+
+import config
+
+conf = config.DBConfig()
+
+
+#Todo(stasya) add logging to except blocks
+class SQLTasks:
+    def __init__(self):
+        self.engine = create_engine('mysql://{user}:{password}@{host}:{port}/highway'.format(
+            user=conf.user, password=conf.password, host=conf.host, port=conf.port),
+            pool_size=2)
+
+        with self.engine.begin() as conn:
+            conn.execute('use highway')
+
+        self.Session = sessionmaker(bind=self.engine)
+
+    def add_task(self, user_id, name, status=None, calendar_date=None):
+        session = self.Session()
+        task = Task(user_id, name, status, calendar_date)
+        try:
+            session.add(task)
+            session.commit()
+            return session.query(Task).filter_by(id=task.id).first()
+        except:
+            session.rollback()
+            raise
+        finally:
+            session.close()
+
+    def update_task(self, task_id, name='', status=None, calendar_date=None):
+        session = self.Session()
+        try:
+            task = session.query(Task).filter_by(id=task_id).first()

```

Comment:
Sorry, but I have a question. Does it work?
here there isn't method update or another query.

And I think it isn't a good idea to ask Database item for changing it.
I mean, these is query for Update.